src/commands/deleteFunctionApp.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// tslint:disable-next-line:no-require-imports
+import WebSiteManagementClient = require('azure-arm-website');
+import { OutputChannel } from 'vscode';
+import { AzureFunctionsExplorer } from '../AzureFunctionsExplorer';
+import { FunctionAppNode } from '../nodes/FunctionAppNode';
+import { getWebSiteClient } from '../nodes/SubscriptionNode';
+
+export async function deleteFunctionApp(explorer: AzureFunctionsExplorer, outputChannel: OutputChannel, node?: FunctionAppNode): Promise<void> {
+    if (!node) {
```

Comment:
verify with user

Reply:
That's done in the shared npm package:
https://github.com/Microsoft/vscode-azuretools/blob/master/appservice/src/SiteWrapper.ts#L99